index.toml

```diff
@@ -227,5 +227,5 @@ notebook = "45_Creating_a_Multi_Agent_System.ipynb"
 aliases = []
 completion_time = "20 min"
 created_at = 2025-06-02
-dependencies = []
+dependencies = ["duckduckgo-api-haystack", "docstring-parser"]
```

Comment:
Should we include `docstring-parser` to the `pip install` cell as well?

Reply:
`docstring-parser` is a default dependency in Haystack starting from 2.15 and it comes by default in colab. I added here for the tests, but we don't need to install it explicitly